source/FallingBlock.hx

```diff
@@ -0,0 +1,80 @@
+package;
+
+import flixel.FlxObject;
+import flixel.FlxSprite;
+import flixel.system.FlxAssets.FlxGraphicAsset;
+import flixel.util.FlxColor;
+import haxe.Timer;
+
+/**
+ * A block that falls after being stepped on
+ * @author TR Nale
+ */
+class FallingBlock extends Block
+{
+	static var FALLACCEL:Float = 100;
+	static var MAXSPEED:Float = 500;
+	static var DESTRUCTIONLINE = 500;
```

Comment:
The note I made earlier about constant, read-only variables also applies to these.